login.html

```diff
@@ -1,48 +1,53 @@
+ <!DOCTYPE html>
+<html>
+<head>
+<link rel = "stylesheet" type = "text/css" href = "login.css">
+</head>
+<body style="position: relative;
+  margin: 0;
+  padding-bottom: 6rem;
+  min-height: 100%;">
 
-
-
-
-
-			
-			
-				
+	 	<div class= "formBox">
+		<form id="InfoBox" action="Evaluation.html">
+			<h3>
```

Comment:
It is a best practice for documents to always include all heading levels leading up to a certain one. In this particular case, it is recommended that this `<h3>` be preceded by a `<h2>`, which itself is preceded by an `<h1>`. That conveys semantic meaning and a logical ordering of the subsections of a document, useful in SEO, among others.